tables/AlternateMans/StokesIStManColumn.h

```diff
@@ -0,0 +1,240 @@
+#ifndef CASACORE_STOKES_I_ST_MAN_COLUMN_H_
+#define CASACORE_STOKES_I_ST_MAN_COLUMN_H_
+
+#include <casacore/tables/DataMan/StManColumn.h>
+
+#include <casacore/casa/Arrays/Array.h>
+#include <casacore/casa/Arrays/IPosition.h>
+
+#include "BufferedColumnarFile.h"
+
+#include <optional>
+
+namespace casacore {
+
+class StokesIStMan;
+
+/**
+ * Expands @p n values from single Stokes I values
+ * to have 4 values, in place. This implies the array
+ * should have place to store n*4 values.
+ */
```

Comment:
Another use case for this storage manager would be to store `WEIGHT_SPECTRUM` (which has four polarizations by default) as one polarization. In that case, the expansion would be to `[value, value, value, value]`. Could you consider making the code generic enough to also handle this?

Reply:
Yes, that's a good point. To make this stman handle both cases, it would have to know if it is a weight col or a data col. This could be determined by the type (double vs complex), by name (`WEIGHT_SPECTRUM` vs anything else) or through the "specification record", but none of these is very elegant. Dysco uses name and type I think, but it has limitations and I don't really like that approach, as it hard-codes some behaviour. 

Given that we also don't need the weight compression at this point, we could ignore the weight scenario for now. After some thinking, I think the most elegant solution is to make a ScalarWeightStMan, and move most of the StokesIStMan code to a base class that these stmans share.

Reply:
(moved)